Bottom: 3a2194cb7df77822b3e14bdf6e2a2aa54cf820ce
Top:    7b720ce9de9142c79338783ff7fba0e74fa22de2
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-28 02:18:31 +0100

macro.h: Remove unused (obsolete) conversion macro TO_PCB


---

diff --git a/src/macro.h b/src/macro.h
index 625d639..1fb6584 100644
--- a/src/macro.h
+++ b/src/macro.h
@@ -50,10 +50,6 @@
 #define	TO_SCREEN_SIGN_X(x)	(SWAP_IDENT ? SWAP_SIGN_X(x) : (x))
 #define	TO_SCREEN_SIGN_Y(y)	(SWAP_IDENT ? SWAP_SIGN_Y(y) : (y))
 
-#ifndef	TO_PCB
-#define	TO_PCB(x)		((LocationType)((x)/Zoom_Multiplier))
-#endif
-
 /* ---------------------------------------------------------------------------
  * misc macros, some might already be defined by <limits.h>
  */
